src/main/java/org/broadinstitute/hellbender/tools/exome/allelefraction/AFModel.java

```diff
@@ -0,0 +1,239 @@
+package org.broadinstitute.hellbender.tools.exome.allelefraction;
+
+import org.apache.commons.math3.special.Gamma;
+import org.broadinstitute.hellbender.tools.exome.AllelicCount;
+import org.broadinstitute.hellbender.utils.mcmc.*;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.IntStream;
+
+import static java.lang.Math.log;
```

Comment:
Just looked at all logarithms in the repo and IMO this code is the  only one that demands the static import for readability.